### DISCUSSION TITLE
Welcome to UMEP Discussions

### DISCUSSION THREAD

**Discussion by biglimp** [UMEP-dev/UMEP#694] [Announcement]
In this forum you can add questions or come up with new ideas for improvements regarding UMEP. Before you start a new discussion, have a look in the history. Maybe someone have already come up with an answer to your specific question.

UMEP Development Team

**Comment by saeed-rayegan** [comment]
Dear UMEP Team,

I’d like to confirm whether SEBE performs its shading and irradiance calculations on an hourly basis using the meteorological file. If so, would it be possible to export hourly irradiance results (for roofs and façades) instead of only annual totals?

This feature would be very useful for coupling SEBE with building energy simulation tools that require hourly solar input.

Thank you for your great work on UMEP and SEBE.

Best regards,
Saeed Rayegan
Concordia University

**Reply by biglimp** [reply]
The SEBE model aggregates the meteorological forcing data to include all data, e.g. a full year, to create a cumulative sky of irradiance (See Lindberg et al. 2015, Solar Energy). So, no, it does not calculate for each hour. Theoretically, it could, if you restart the model at an hourly resolution but this is not the intent of this model.
